src/DynamoCoreWpf/ViewModels/Core/Converters/DynamoTextOptions.cs

```diff
@@ -0,0 +1,37 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Dynamo.Wpf.ViewModels.Core.Converters
+{
+    public class DynamoTextOptions
+    {
+        private static ICSharpCode.AvalonEdit.TextEditorOptions TextOptions;
+
+        /// <summary>
+        /// Class to interact with ICSharp Text Editor Options
+        /// </summary>
+        public DynamoTextOptions()
+        {
+            if(TextOptions == null)
+                TextOptions = new ICSharpCode.AvalonEdit.TextEditorOptions();
+        }
+        /// <summary>
+        /// Sets the text editor options
+        /// </summary>
+        public void SetTextOptions(bool value) {
```

Comment:
I think this should be very clear since it's public - 

1. let's call it `ShowWhiteSpaceCharacters` etc
2. it needs /// summary and parameter tags / comments